src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -16,20 +21,59 @@ function createOrReplaceSite(args, res) { // eslint-disable-line no-unused-vars
  * @returns {Promise.<{runTime: string, pages: Array<{pageUrl: string, RowKey: string}>}>}
  */
 function modifyFacebookPages(args, res) { // eslint-disable-line no-unused-vars
+  return new Promise((resolve, reject) => {
+    reject('Modification not yet supported.');
+  });
 }
 
 /**
  * @param {{input: {site: string, pages: Array<{pageUrl: string, RowKey: string}>}}} args
  * @returns {Promise.<{runTime: string, pages: Array<{pageUrl: string, RowKey: string}>}>}
  */
 function removeFacebookPages(args, res) { // eslint-disable-line no-unused-vars
+  return new Promise((resolve, reject) => {
+    reject('Removal not yet supported.');
+  });
+}
+
+function trustedTwitterAccountRowKeyToPrimaryKey(account) {
+  const params = account.RowKey.split(/,/);
```

Comment:
Nit: can just split this by string instead of regex since it's a simple split: `account.RowKey.split(',')`